### DISCUSSION TITLE
Error inviting user

### DISCUSSION THREAD

**Discussion by abhibollam** [Budibase/budibase#9780] [Help]
Hi team,

can some one please help on this

I'm getting this error when I try to create a new user through the web app. i'm not able to create a single user

i have attached the screen shots of logs for bbworker 

![Screenshot 2023-02-23 021302](https://user-images.githubusercontent.com/126145059/220841144-99c11574-6e57-416b-8a2c-a8be114cff46.png)

![image](https://user-images.githubusercontent.com/126145059/220841277-f5cb91f8-eb0a-4d24-a5e9-874682772d01.png)

**Comment by 100wunder** [comment]
I had the same problem. Eventually it worked with another email address that was not already registered at budibase.

**Reply by abhibollam** [reply]
for me it's not allowing to add users at all, whatever email i give it fails

**Comment by andz-bb** [comment]
hey @abhibollam, sorry I'm only responding to this now. Can you check your browser's developer tools console and network tab (F12 in Chrome) whenever you try to add a user? There may be an error there that'll help me to understand what's going on.

**Reply by abhibollam** [reply]
https://github.com/Budibase/budibase/discussions/9780#discussioncomment-5092123

**Comment by abhibollam** [comment]
these are the errors i got 

![image](https://user-images.githubusercontent.com/126145059/221002889-e6e7d432-79de-4207-b171-23e5bda13440.png)
![image](https://user-images.githubusercontent.com/126145059/221003176-6bee3651-d60b-4770-9cdb-641bb2ec1efd.png)
